src/test/java/RMSysTest.java

```diff
@@ -0,0 +1,96 @@
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.*;
+
+public class RMSysTest extends TestBase {
+
+    @Test
+    public void fillingLoginForm() {
+        open("https://192.168.100.26/");
+        $("#Username").setValue("EugenBorisik");
+        $("#Password").setValue("qwerty12345");
+        $("#SubmitButton").click();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertNotNull(".sign-out-span>a");
+    }
+
+    @Test
+    public void explicitCondTest() {
+        open("https://192.168.100.26/");
+        $("#Username").setValue("EugenBorisik").shouldBe(visible);
+        $("#Password").setValue("qwerty12345").shouldBe(visible);
+        $("#SubmitButton").click();
+        $("#officeMenu").shouldBe(visible).click();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertNotNull(".sign-out-span>a");
+
+    }
+
+    @Test(dataProvider = "IncorrectDataSets", dataProviderClass = DataProvider.class)
+    public void InvalidCredsDDTTest(String username, String password) {
+        open("https://192.168.100.26/");
+        $("#Username").setValue(username);
+        $("#Password").setValue(password);
+        $("#SubmitButton").click();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals($(".validation-summary-errors li"), "*Invalid credentials.");
+    }
+
+    @Test(dataProvider = "IncorrectTestUsers", dataProviderClass = DataProvider.class)
+    public void PasswordRequiredDDTTest(String username, String password) {
+        open("https://192.168.100.26/");
+        $("#Username").setValue(username);
+        $("#Password").setValue(password);
+        $("#SubmitButton").click();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals($("#password-box-validation>span"), "Password is required");
```

Comment:
This code doesn't check anything. Please ask me, if it's not clear why this code is senseless
By the way, replace this code with Selenide should() method